metron-analytics/metron-profiler/src/main/assembly/assembly.xml

```diff
@@ -63,7 +63,7 @@
         <fileSet>
             <directory>${project.basedir}/target</directory>
             <includes>
-                <include>${project.artifactId}-${project.version}.jar</include>
+                <include>${project.artifactId}-${project.version}-uber.jar</include>
             </includes>
```

Comment:
Is it important to change the jar names? If so, could you check for ripple changes to the RPM build scripts and the Ambari scripts?